Forms/Schema/FormSchema.php

```diff
@@ -67,10 +67,10 @@ public function getState(Form $form) {
 			$this->getFieldStates($form->Actions())
 		);
 
-		if($form->Message()) {
-			$state['messages'][] = [
```

Comment:
Agreed - multiple messages for a form is likely useful, but multiple messages for form fields should be inlined into a single message, or showing the most important message first (like on the http://redux-form.com/6.1.1/examples/syncValidation/ example: `((error && <span>{error}</span>) || (warning && <span>{warning}</span>))`)